test/build/__snapshots__/build.test.js.snap

```diff
@@ -4631,6 +4631,11 @@ exports[`snowpack build package-tippy-js: web_modules/import-map.json 1`] = `
 
 exports[`snowpack build package-tippy-js: web_modules/tippyjs.js 1`] = `
 "import { c as createPopper } from './common/popper-XXXXXXXX.js';
+/**!
+* tippy.js v6.2.6
```

Comment:
huh, this is unexpected. Not necessarily bad, just suprising

Reply:
Yeah, I wasn't too sure what to make of it either.  My only guess is that something (?) now knows statically that we're not making a production build, so it's keeping the comment there.